tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
How does HQ know (when it's constructing the schema) that

child > mother > household

is an acceptable way to navigate to a grandparent (household), but

child > mother > first-child

is not?